cmd/tsbs_generate_queries/databases/influx/devops_test.go

```diff
@@ -114,3 +360,122 @@ func TestDevopsFillInQuery(t *testing.T) {
 		t.Errorf("filled query has wrong path: got %s want /query?%s", got, encoded)
 	}
 }
+
+type Case struct {
```

Comment:
i know it ultimately doesn't matter, but maybe lowercase this so someone doesn't get confused about its exported-ness

Reply:
Make sense, I'm drawing a blank as to why I made it exported in the first place.